ocp4crd/oauth/oauth.go

```diff
@@ -0,0 +1,135 @@
+package oauth
+
+import (
+	"log"
+
+	configv1 "github.com/openshift/api/legacyconfig/v1"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	"gopkg.in/yaml.v2"
+
+	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func init() {
+	configv1.InstallLegacy(scheme.Scheme)
```

Comment:
This one is not needed here. Though let it be here for now and remove in future commits.